internal/syncronizedmap/syncronizedmap.go

```diff
@@ -0,0 +1,54 @@
+package syncronizedmap
+
+import "sync"
+
+//FIXME: add UT
+type SyncronizedKernelOsDtkMapping interface {
+	NewNodeInfo(kernelVersion, osImage string)
+	NewImageStreamInfo(osImage, dtkImage string)
+	GetImage(kernelVersion string) (string, bool)
+}
+
+type syncronizedKernelOsDtkMapping struct {
+	m          *sync.RWMutex
+	kernelToOs map[string]string
+	osToDtk    map[string]string
```

Comment:
Capitalized words can remain capital in variable names:
```suggestion
	kernelToOS map[string]string
	osToDTK    map[string]string
```

Reply:
I think it is clearer that way. If I change it, I will also change it in other places and sometimes is is more confusing than helping.

For exampel:
`SyncronizedKernelOsDtkMapping` vs `SyncronizedKernelOSDTKMapping`